Content.Shared/Info/SharedRulesManager.cs

```diff
@@ -23,4 +24,36 @@ public override void WriteToBuffer(NetOutgoingMessage buffer)
             buffer.Write(PopupTime);
         }
     }
+
+    /// <summary>
+    ///     Sent by the server when the client needs to display the rules on join.
+    /// </summary>
+    public sealed class ShouldShowRulesPopupMessage : NetMessage
+    {
+        public override MsgGroups MsgGroup => MsgGroups.Command;
```

Comment:
This shouldn't be a `Command` message because of ordering issues but basically the entire netcode is a mess when it comes to this so I'll let it slide.